ember-bootstrap/addon/components/bs-modal/footer.hbs

```diff
@@ -8,7 +8,7 @@
         <Button
           @type={{bs-default @submitButtonType "primary"}}
           @onClick={{@onSubmit}}
-          {{bs-conditional-attribute "disabled" @submitDisabled @submitDisabled}}
+          {{bs-conditional-attribute "disabled" @submitDisabled (if @submitDisabled "true" "false")}}
```

Comment:
This seems to be wrong. The value is only taken into consideration if the condition is true. The condition is `@submitDisabled`. The value can be an empty string for boolean attributes such as `disabled`.

Reply:
Ah, you're right. I was too preoccupied with looking for a toString inside the template that I overlooked that it only uses the "true" value anyway